src/tooltip/test/tooltip.spec.js

```diff
@@ -277,6 +278,52 @@ describe('tooltip', function() {
 
   });
 
+  describe('with specified popup close delay', function () {
+
+    beforeEach(inject(function ($compile) {
+      scope.delay='1000';
+      elm = $compile(angular.element(
+        '<span tooltip="tooltip text" tooltip-popup-delay="1000"' +
+        'tooltip-popup-close-delay="{{delay}}">Selector Text</span>'
+      ))(scope);
+      elmScope = elm.scope();
+      tooltipScope = elmScope.$$childTail;
+      scope.$digest();
+    }));
+
+    it('should open after timeout and close after the close delay timeout', inject(function ($timeout) {
+
+      elm.trigger('mouseenter');
```

Comment:
I was not aware that it was from jQuery. On that note, I think we should create an issue to not use the ```.trigger```. I am most definitely all aboard for removing jQuery as a dependency (I was also unaware that was a goal in the link you provided). 

Thank you for your posts, I always appreciate the opportunity to learn more. If you agree with an issue being created to re-write the tests that are utilizing jQuery, I would be more than happy to create them and then update this file to use ```.triggerHandler()``` as you've suggested.